website/README.md

```diff
@@ -4,6 +4,8 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 
 ## Installation
 
+In a terminal, cd into the website folder of the locally cloned unleash repository. Then execute:
```

Comment:
Maybe we should add the command for changing directories too? 

Something like this, maybe?

```suggestion
In a terminal, cd into the website folder of the locally cloned unleash repository. 

```console
cd unleash/website
```

Then install the dependencies:
```